web/src/features/fbaCalculator/components/GrassCureCell.tsx

```diff
@@ -54,11 +48,12 @@ const GrassCureProps = (props: GrassCureCellProps) => {
   }
 
   const hasError = grassCureNotSetForGrassType(
-    props.fbcInputGridProps.inputRows[props.rowId]
+    props.fbaInputGridProps.inputRows[props.rowId]
   )
 
   return (
     <TextField
+      data-testid={`grassCureInput-${props.rowId}`}
       type="number"
       className={props.classNameMap.grassCure}
       size="small"
```

Comment:
the `TextField` element in `WindSpeedCell` has an additional attribute `inputMode="numeric"`, which means that if users are using a mobile device to access the app, when they click into the cell, the number keyboard gets automatically rendered instead of the alphabetical one. Probably not very important because I don't think many (or maybe not any) of our users are accessing FireBat on mobile devices, but might be a nice-to-have just in case?

Reply:
Added `inputMode="numeric"` property to `GrassCureCell`.